exclude :test_chain_each_lambda, "<true> expected but was <false>."
exclude :test_chain_undef_methods, "<0> expected but was <5>."
exclude :test_each_arg, "<nil> expected but was <1>."
exclude :test_each_slice_size, "FloatDomainError: Infinity"
exclude :test_generator, "[FrozenError] exception expected, not #<ArgumentError: wrong number of arguments (given 1, expected 0)>."
exclude :test_initialize, "ArgumentError expected but nothing was raised."
exclude :test_initialize_copy, "TypeError expected but nothing was raised."
exclude :test_inspect, "<\"#<Enumerator: 0..10:each_with_object(a: 1)>\"> expected but was <\"#<Enumerator: 0..10:each_with_object({:a=>1})>\">."
exclude :test_lazy_chain, "hangs"
exclude :test_peek_modify, "<[1, 2]> expected but was <[1, 2, 3]>."
exclude :test_peek_values_modify, "<[1, 2]> expected but was <[1, 2, 3]>."
exclude :test_size, "TypeError expected but nothing was raised."
exclude :test_size_for_cycle, "<0> expected but was <Infinity>."
exclude :test_size_for_downup_to, "<0> expected but was <nil>."
exclude :test_stopiteration_rescue, "Expected \"<internal:core> core/enumerator.rb:164:in `next'\" to include \"test_enumerator.rb:251:\"."
exclude :test_uniq, "<[0, 1]> expected but was <[]>."
